index.html

```diff
@@ -1504,57 +1504,69 @@
-have an interest in knowing which sites that they link to [=people=] choose to visit (which in turn
-could improve the service for everyone). But [=people=] may have an interest in keeping that
-information private from algorithmic media companies (as do the sites being linked to, as that
-facilitates timing attacks to recognise [=people=] there). A [=person=]'s exit through a specific
-link can either be tracked with JavaScript tricks or through bounce tracking, both of which are slow
-and difficult for user agents to defend against. The value proposition of the [^a/ping^] attribute
-in this context is therefore straightforward: by providing declarative support for this
-functionality it can be made fast (the browser sends an asynchronous notification to a ping
-endpoint after the [=person=] exits through a link) and the user agent can provide its [=user=] with
-the option to opt out of such tracking — or disable it by default.
-
-Unfortunately, this arrangement proved to be unworkable on the privacy side (the performance gains,
-however, are real). What prevents a site from using [^a/ping^] for [=people=] who have it activated
-and bounce tracking for others? What prevents a browsers from opting everyone out because it wishes
+Even though collective decision-making can be better than offloading
+[=privacy labour=] to [=individuals=], it is not a panacea.
+Decision-making bodies need to be designed carefully,
+for example using the <a data-cite="IAD#">Institutional Analysis and Development framework</a>.
+
+<aside class="example" id="example-a-ping">
+
+The standardisation of the [^a/ping^] attribute serves as a good example of a failure in collective privacy decisions.
+Search engines, social sites, and other algorithmic websites
+have an interest in knowing which of their links their visitors choose to click on.
+They can use this information to improve their service for everyone.
+But their visitors may have an interest in keeping that information private,
+for example because it can be used to [=recognize=] those visitors across [=partitions=].
+
+A website can track a [=person=]'s click on a specific link
+using either JavaScript {{HTMLElement/onclick}} events or bounce tracking,
+both of which are slow and difficult for user agents to defend against.
+The [^a/ping^] attribute declares the site's intent,
+which lets a user agent either implement it quickly and asynchronously
+or ignore it for users who want to keep that information private.
+
+Unfortunately, while this arrangement did improve performance,
+it turned out not to help privacy.
+What prevents a site from using [^a/ping^] for [=people=] who have it activated
+and bounce tracking for others? What prevents a browser from opting everyone out because it wishes
 to offer better protection by default? Given the contested nature of the [^a/ping^] attribute and
 the absence of a forcing function to support collective enforcement, the scheme failed to deliver
 improved privacy.
```

Comment:
I wasn't sure exactly what we meant here by "a forcing function to support collective enforcement", so I haven't changed it to simpler wording. What should get enforced? That websites exclusively use `ping` instead of bounce tracking?

I'm also not sure that we're right that `<a ping>` didn't help privacy. My understanding is that [there are tracking-blocking extensions that do block the network requests from the `ping` attribute](https://github.com/gorhill/uBlock/wiki/Dashboard:-Settings#disable-hyperlink-auditing), and that websites have some trouble falling back to bounce tracking in that case. Is there something we can cite that'll better demonstrate that there's a problem?